### PR TITLE
Instances rules modifications

### PR DESCRIPTION
**Pull request by SamantazFox**
Instances rules modification proposal as a follow-up of the discussions from #148 and #149.

The idea is to provide an ad/tracker free official list, and to clarify some other rules.

notes: it could be a good idea to also remove the 1st rule and transform it to a more meaningful note/warning saying that the newly added instances will be on a probatory period of 1 months (should be more imho) before they can be added to the official list, so we can ensure that the owner is able to keep it updated at least once a month.

### CODE REVIEW

File: Invidious-Instances.md [hunk 1]
```diff
@@ -69,22 +69,12 @@ dateCreated: 2021-05-23T16:58:48.431Z
...
 7. Instances using any DDoS Protection / MITM MUST be marked as such (eg: Cloudflare, DDoS-Guard...).
 8. Instances using any type of anti-bot protection MUST be marked as such.
-9. Instances using any type of analytics:
-   - MUST be marked as such
-   - MUST be GDPR compliant
-   - MUST be CCPA compliant
-   - Are, by essence, running a modified source code, so the next points applies too
-10. Any system whose goal is to modify the content served to the user (HTTP server rewrite, scripts, etc...) is considered the same as modifying the source code.
+9. Instances MUST NOT use any type of analytics.
+9. Instances MUST NOT serve ads (sponsorship links in banner are considered ads) NOR promote products.
```
[the remark this answers is not in the document]
Comment (syeopite) [reply Invidious-Instances.md:73]: Not a big fan of that as well but it seems to be the consensus reached in #149

[the remark this answers is not in the document]
Comment (SamantazFox) [reply Invidious-Instances.md:73]: > You want to disallow sponsorship completely? I'm personally not a fan in enforcing that, but I guess it's fair.

I was quite neutral about it, but as @syeopite says, it's what the majority of the current instance owners  agreed upon. And honnestly, if we forbid ads, we should forbid sponsoring too; first, to be consistent, but also to not have to bother trying to draw a line between the two.

**Review by TheFrenchGhosty — APPROVED** [review]

File: Invidious-Instances.md [hunk 2]
```diff
@@ -69,22 +69,12 @@ dateCreated: 2021-05-23T16:58:48.431Z
...
     - MUST publish any later modification in a timely manner
     - MUST contain a link to both the modified and original source code of Invidious in the footer.
-12. Instances are free to serve ads or promote sponsors but they:
-    - MUST be marked as such
-    - MUST NOT be served from a third party (eg: Google Ads isn't allowed)
-    - MUST NOT be scam
-    - MUST be for 100% legal services/goods
-    - MUST be explicitly marked as "not endorsed in any way by the Invidious project"
-13. Instances MUSTN'T promote anything illegal (even if it's not advertisement).
+12. Instances MUST NOT serve ads (sponsorship links in banner are considered ads) NOR promote products.
```
Comment (TheFrenchGhosty) [inline Invidious-Instances.md:78]:
```suggestion
12. Instances MUST NOT serve ads (sponsorship links in the banner are considered ads) NOR promote products.
```

**Review by syeopite — CHANGES_REQUESTED** [review]
I think this should be reformatted to look more like our issue template for new instances, and less like a wall of text. This gigantic list makes it looks like we have far more rules (and in turn more controlling) then we actually do.

**Pull request merged**